cpp/ql/src/semmle/code/cpp/rangeanalysis/PointlessComparison.qll

```diff
@@ -11,8 +11,17 @@ private float lowerBoundFC(Expr expr) { result = lowerBound(expr.getFullyConvert
 /** Gets the upper bound of the fully converted expression. */
 private float upperBoundFC(Expr expr) { result = upperBound(expr.getFullyConverted()) }
 
+/**
+ * Describes which side of a pointless comparison is known to be smaller.
+ */
 newtype SmallSide =
+  /**
+   * Represents that the left side of a pointless comparison is known to be smaller.
```

Comment:
"Represents that" seems unnecessary. I would think that an "enum" IPA type would just need its branches to be commented with something like "The left side of the pointless comparison is known to be smaller.". Does that sound any better?

Reply:
The current wording is awkward, but I felt like it was necessary for the VS Code mouseover display, where the branch QLDoc shows up by itself.

Reply:
SGTM.